### DISCUSSION TITLE
Filtering rack elevations to specific racks

### DISCUSSION THREAD

**Discussion by DenHayes** [netbox-community/netbox#8797] [Ideas]
Has any one looked into adding the rack field to the Rack Elevations filter?  

It would be nice to be able to filter the elevations to specific racks, so you can view a subset of racks within a site or location.

**Comment by jeremystretch** [comment]
FYI you can do this by manually appending the `id` filter a number of times. For example: https://demo.netbox.dev/dcim/rack-elevations/?id=39&id=40 (However, it's not built into the UI.)

**Reply by DenHayes** [reply]
That's fine as a work around. Having to look up the rack ids is a bit of a pain long term.

Are there others out that there that see this as benefit? If so, I can submit feature request.

**Reply by hchybz** [reply]
> FYI you can do this by manually appending the `id` filter a number of times. For example: https://demo.netbox.dev/dcim/rack-elevations/?id=39&id=40 (However, it's not built into the UI.)

in v2, it was pobbislbe to filter elevation with rack names.. why did you remove this feature in v3 ?

**Comment by lastorel** [comment]
Yeah that would be useful to visually check the space in any location/site
